packages/core/src/+internal/utils/array.util.ts

```diff
@@ -1,2 +1,13 @@
+import { Maybe } from '../fp/maybe';
+
 export const getArrayFromEnum = (E: Object) =>
   Object.keys(E).filter(key => typeof E[key as any] === 'number');
+
+export const getHead = <T>(array: T[]) =>
+  Maybe.of(array[0]);
+
+export const filterArray = <T>(f: (v: T) => boolean) => (array: T[]) =>
```

Comment:
You use it only in the test. Do you want to keep this function for future features?